.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,16 +17,30 @@ Reviewing guidelines: https://hydra.nixos.org/job/nixpkgs/trunk/manual/latest/do
 
 ###### Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- 
+Requirements: 
```

Comment:
I am not convinced we should make `nixpkgs-preview` a hard requirement at this time. It is [too much memory](https://github.com/Mic92/nixpkgs-review/issues/103) so running it is not an option for some reviewers. (I can only run it if I quit Firefox, or my computer freezes when trying to swap :crying_cat_face:)